BREEZE.rst

```diff
@@ -1,2170 +0,0 @@
- .. Licensed to the Apache Software Foundation (ASF) under one
```

Comment:
I guess deleting BREEZE.rst was not intentional @frncescomucio :) . It could have been some side effect of a pre-commit updating Breeze's --help-all output, and it would be great to find out how to find out how it has happened.

Reply:
I think I removed it with `git -rm` to remove some unintended changes. Not very smart

Reply:
:D